content/spin/v1/ai-sentiment-analysis-api-tutorial.md

```diff
@@ -174,7 +174,8 @@ $ mv llama-2-13b-chat.ggmlv3.q3_K_L.bin llama2-chat
 tree .spin
 .spin
 └── ai-models
-    └── llama2-chat
+    └── llama
```

Comment:
This is not needed for `llama2-chat`.  We should make a decision on whether we want to recommend always using the `llama` directory or only using it for "custom" models.